app/views/content/is-teaching-right-for-me/career-progression.md

```diff
@@ -0,0 +1,132 @@
+
+keywords:
+  - Professional Development
+  - Teacher Promotion
+  - Career Development
+  - Progression
+  - Career Progression
+  - Development
+  - Scale
+  - Teacher Salary
+  - Teacher Salaries
+  - Leading Practitioner
+  - Headteacher
+  - Progression
+  - Promotion
+  - Teaching and learning responsibility
+  - Teaching and learning responsibility payment
+  - TLR
+  - Teaching Roles
+  - Promoted
+  - Teaching Responsibilities
+  - Additional
+  - Payments
+  - Responsibility
+  - Responsibilities
+  - Career
+  - Career Prospects
+  - Benefits
+  - Earnings
+  - Leadership
+  - NQT
+  - National professional qualification
+  - Support
+  - Training
+  - Salary increase
+  - Salary review
+  - Pastoral care
+  - ECF
+  - Teaching
+  - Teacher training
+  - How to become a teacher
+  - Teacher training courses
+---
+
+Teacher training advisers Chris Farr, Fran Lee, Sian Mills and Elaine Boyd have years of teaching experience. They talk about teacher development and how you can progress a career in teaching.
+
+## Is there a set path to getting promoted as a teacher?
+
+<strong>Fran:</strong> Your career prospects can vary vastly depending on where you’re located, your employer and your training experience. For example, you might decide in your training period to set your targets in a particular area such as special educational needs and disabilities (SEND), or you might work in a sixth-form environment. And that can sway where your future takes you.  
+
+<strong>Sian:</strong> When I worked at a maintained primary state school it was quite traditional in terms of class teacher, head of year, deputy/assistant then headteacher. When I moved to a headship in an academy trust there was no deputy head. So you might not always go into a school that has a traditional structure. There’s not always a neat pathway.
+
+<strong>Chris:</strong> People may not realise there is much progression in teaching. They think they go into the world of the classroom and that’s it. Lots of teachers have added responsibilities they enjoy doing and find fulfilling. 
+
+## Can you change the subject you teach?
+
+<strong>Fran:</strong> I've had people wanting to know if they train in geography can they move out of that subject area or become head of year. Once you get qualified teacher status (QTS) you can essentially teach anything as long as your headteacher supports you and you have the expertise to show you can do this. 
+
+## What are teacher pastoral care roles?
+
+<strong>Fran:</strong>  Pastoral care in schools is looking at nurturing pathways. It’s about recognising all children have a right to feel safe and happy in school and guiding children on a more personal level. A [special educational needs co-ordinator (SENCO)](/teach-disabled-pupils-and-pupils-with-special-educational-needs) is part of pastoral care. Or you might be a form tutor and look after your pupils’ emotional wellbeing. 
```

Comment:
```suggestion
<strong>Fran:</strong>  Pastoral care in schools is looking at nurturing pathways. It’s about recognising all children have a right to feel safe and happy in school and guiding children on a more personal level. A [special educational needs co-ordinator (SENCO)](/is-teaching-right-for-me/teach-disabled-pupils-and-pupils-with-special-educational-needs) is part of pastoral care. Or you might be a form tutor and look after your pupils’ emotional wellbeing. 
```

This is failing a spec test as the page is not found (although the link works because this happens to be a redirect!)